files/config/kube-proxy.yaml

```
@@ -11,3 +11,4 @@ conntrack:
   maxPerCore: {{ .Cluster.Kubernetes.NetworkSetup.KubeProxy.ConntrackMaxPerCore }}
 {{- end }}
 metricsBindAddress: 0.0.0.0:10249
+bind-address-hard-fail: true
```

This is the flag name but the config file has a different naming convention https://kubernetes.io/docs/reference/config-api/kube-proxy-config.v1alpha1/

```suggestion
bindAddressHardFail: true
```